lustrefs-exporter/src/jobstats.rs

```diff
@@ -70,57 +70,64 @@ pub fn jobstats_stream<R: BufRead + std::marker::Send + 'static>(
 ) -> (JoinHandle<Result<(), Error>>, Receiver<CompactString>) {
     let (tx, rx) = mpsc::channel(200);
 
-    let x = tokio::task::spawn_blocking(move || {
-        let mut state = State::Empty;
+    let x = tokio::task::spawn(async move {
+        tokio::task::spawn_blocking(move || {
+            let mut state = State::Empty;
```

Comment:
It allows to return a `JoinHandle` and await the blocking task in it.
@jgrund if you have a better idea

Reply:
I'll push something, one sec...